kombu/asynchronous/aws/sqs/connection.py

```diff
@@ -58,7 +58,7 @@ def set_queue_attribute(self, queue, attribute, value, callback=None):
 
     def receive_message(self, queue, queue_url,
                         number_messages=1, visibility_timeout=None,
-                        attributes=None, wait_time_seconds=None,
+                        attributes=('ApproximateReceiveCount',), wait_time_seconds=None,
```

Comment:
what is ('ApproximateReceiveCount',) doing actually?

Reply:
ApproximateReceiveCount is a parameter requested to be fetched from AWS SQS, returning the number of re-drives the specific message went through. 

`ApproximateReceiveCount – Returns the number of times a message has been received across all queues but not deleted`

Reply:
will check this thoroughly tomorrow morning.

Reply:
@auvipy I created the following project demonstrating the change https://github.com/galCohen88/celery_sqs_retry_policy

Reply:
I am not expert in SQS to be honest. We are changing default value of attribute. Is this breaking change for the users?

Reply:
I think since its a new feature we should be OK with that.

Reply:
In its response to ReceiveMessage, SQS can provide an (optional) attribute called ApproximateReceiveCount.

This attribute allows for rudimentary diagnostics of retries resulting from worker termination (assuming late ACKs).